server/src/main/java/org/candlepin/model/Consumer.java

```diff
@@ -762,4 +762,14 @@ public void setContentAccessMode(String contentAccessMode) {
         this.contentAccessMode = contentAccessMode;
     }
 
+    /**
+     * This will put in the property so that the virtWho instances won't error
+     *
+     * @return List always empty
+     */
+    @JsonProperty("guestIds")
+    public List getEmptyGuestIdArray() {
```

Comment:
(duplicate comment, sorry)
Are we ever going to use this property going forward? If so, does it make sense to just call this "getGuestIDs" and have it always return an empty array until we need it to do otherwise? Maybe the doc block is enough documentation here.